webapp/src/app/components/toolbar/toolbar.component.html

```diff
@@ -2,31 +2,42 @@
 <mat-toolbar fxLayout="row" class="mat-elevation-z4 elevated-toolbar">
 	
 	<div fxFlex fxLayoutAlign="start">
-		<button mat-button [matMenuTriggerFor]="file">File</button>
-		<mat-menu #file="matMenu">
-			<button mat-menu-item *ngIf="loaded" (click)="loadMapClicked.emit()">Load Map</button>
-			<button [disabled]="!map" mat-menu-item (click)="saveMap(false)">Save Map (Ctrl + S)</button>
-			<button [disabled]="!map" mat-menu-item (click)="saveMap(true)">Save Map as... (Ctrl + Shift + S)</button>
-		</mat-menu>
+		<div [style.visibility]="is3d ? 'hidden': 'visible'">
```

Comment:
I think changing this to something that doesn't take up space when hidden might look better. Open for opinions.

Reply:
It would look better on it's own, but without the space the 3d toggle would move which is not a good user experience